src/modules/identity/RadixRemoteIdentity.ts

```diff
@@ -0,0 +1,108 @@
+import RadixECIES from '../crypto/RadixECIES'
+import RadixIdentity from './RadixIdentity'
+import RadixAccount from '../account/RadixAccount'
+import RadixSignature from '../atom/RadixSignature'
+import RadixSerializer from '../serializer/RadixSerializer'
+
+import { RadixAtom, RadixKeyPair } from '../RadixAtomModel'
+
+export default class RadixRemoteIdentity extends RadixIdentity {
+    private TOKEN: string
+    private REMOTE_URL: string
```

Comment:
Now, I'm really humble for my own ignorance here: In Java we only use capitalised member names for constants (e.g. `public static final int DEFAULT_PORT = 10000;` as opposed to variables; `private int itemCount;`). Is there a similar "de facto standard" in JS? Is it customary to use capitalised variable names in general?

Reply:
With you on this one - if these were hardcoded constants, I'd have them capitalised, but they're being set in the constructor.

Reply:
@laszlourszuly there's not one standard per se in Javascript, just many accepted ways of doing the same thing. But I agree, it's confusing to use uppercase so I'll change them.